src/react/account/details/properties/__tests__/AccountInfo.test.tsx

```diff
@@ -159,4 +158,33 @@ describe('AccountInfo', () => {
       return expect(screen.getByText('Account Page')).toBeInTheDocument();
     });
   });
+
+  it('should display an error message when attempting to delete if there is a bucket attached to the account', async () => {
+    //S
+    server.use(
+      rest.delete(
+        `${TEST_API_BASE_URL}/api/v1/config/${INSTANCE_ID}/user`,
+        (_, res, ctx) => res(ctx.status(409)),
+      ),
+    );
+    //E
+    renderWithRouterMatch(<AccountInfo account={account1} />, undefined, {
+      instances: { selectedId: INSTANCE_ID },
+    });
+    userEvent.click(screen.getByRole('button', { name: /Delete Account/i }));
+    userEvent.click(
+      within(screen.getByRole('dialog', { name: /Confirmation/i })).getByRole(
+        'button',
+        { name: /delete/i },
+      ),
+    );
+    //V
+    await waitFor(() => {
+      expect(
+        screen.getByText(
+          /Unable to delete the account due to the presence of associated resources./i,
+        ),
+      ).toBeInTheDocument();
```

Comment:
Shall we ensure the text to be within the modal ?

```suggestion
      expect(
        within(screen.getByRole('dialog', { name: /Confirmation/i })).getByText(
          /Unable to delete the account due to the presence of associated resources./i,
        ),
      ).toBeInTheDocument();
```